src/Ariel/Syntax/Parse.hs

```diff
@@ -145,17 +155,24 @@ parseTermBinding name = do
 
 -- EBNF:
 -- PrimaryExpr ::= LetExpr
+--               | LetRecExpr
 --               | LambdaExpr
 --               | identifier
 --               | Literal
 --               | '(' Expr ')'
+--               | Tuple
 --
 -- ExprList ::= '(' (Expr ',')* ')'
 --
 -- Term ::= PrimaryExpr (ExprList)*
 --
 -- Expr ::= Expr identifier Expr
 --        | Term
+--
+-- Tuple ::= '{' (PrimaryExpr ',')* '}'
+-- 
+-- LetExpr :: 'let' Decl 'in' Expr
+-- LetRecExpr :: 'let' 'rec' Decl 'in' Expr
```

Comment:
Let's change the `in` back to `,`.

Reply:
ah yes i forgot about that, ill fix that really quick